### PR TITLE
LPS-142510 Upgrade graphiql version

### PR DESCRIPTION
**Pull request by ccorreagg**
Related ticket: [LPS-142510](https://issues.liferay.com/browse/LPS-142510)

This PR contains the changes to upgrade the graphiql version to remove [this vulnerability](https://nvd.nist.gov/vuln/detail/CVE-2021-41248).

Furthermore, when upgrading the version an error shows up by building the GraphQL schema when creating an empty ObjectType to cover Objects functionality where there is non created objects or all of them are unactive.

**With the upgraded version without the validation**

_Without any Object or any active Object:_

image

_With at least one Object active:_

image

**With the upgraded version with the validation**

_Without any Object or any active Object:_

image

_With at least one Object active (called 'alumno'):_

image

### CODE REVIEW

**Comment by ccorreagg** [comment]
ci:stop

**Comment by ccorreagg** [comment]
ci:test:relevant

**Comment by ccorreagg** [comment]
ci:test:sf

File: modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/graphql/servlet/GraphQLServletExtender.java [hunk 1]
```diff
@@ -2045,6 +2046,13 @@ private void _registerGraphQLDTOContributors(
 		GraphQLObjectType.Builder rootMutationGraphQLObjectTypeBuilder,
 		GraphQLObjectType.Builder rootQueryGraphQLObjectTypeBuilder) {
 
+		Collection<GraphQLDTOContributor> graphQLDTOContributors =
```
Comment (javierdearcos) [inline modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/graphql/servlet/GraphQLServletExtender.java:2049]: 👏 👏 👏 

That was the culprit of the error... Makes total sense.

**Comment by javierdearcos** [comment]
ci:forward

**Pull request closed without merging**